apps/wing-console/console/ui/src/layout/default-layout.tsx

```diff
@@ -43,6 +44,9 @@ const defaultLayoutConfig: LayoutConfig = {
   },
   bottomPanel: {
     components: [
+      {
```

Comment:
I don't think it should be part of the bottom panel, it doesn't look good.
For me it make more sense to add it to the leftPanel components (above the tests).

![Screenshot 2023-12-24 at 14 01 32](https://github.com/winglang/wing/assets/2538825/3e48880c-4d0f-4178-9ab8-b5124c5c084b)